tests/e2e/test_rosacli_cluster.go

```diff
@@ -332,6 +332,145 @@ var _ = Describe("Edit cluster",
 					To(ContainSubstring(
 						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))
 			})
+
+		// Excluded until bug on OCP-73161 is resolved
+		It("can verify delete protection on a rosa cluster - [id:73161]",
+			labels.High, labels.Runtime.Day2, labels.Exclude,
+			func() {
+				By("Enable delete protection on the cluster")
+				deleteProtection := constants.DeleteProtectionEnabled
+				_, err := clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=true",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check the enable result from cluster description")
+				output, err := clusterService.DescribeCluster(clusterID)
```

Comment:
You can use directly DescribeandReflectClusterDescription function , rather than using two functions Describe and ReflectClusterDescription function individually

Reply:
Changed